src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunMinimalBWAMEMTest.java

```diff
@@ -0,0 +1,91 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Tests RunMinimalBWAMEM.
+ */
+public final class RunMinimalBWAMEMTest extends CommandLineProgramTest {
+
+    private static final File TEST_DATA_DIR = new File(getTestDataDir(), "spark/sv/RunMinimalBWAMEM");
+
+    @Test(groups="sv")
+    public void testSeparate() throws IOException {
+
+        try{
+            CMDLineProgramModule.checkIfProgramIsAvailableOnHost("bwa");
+        } catch(final IOException e){
+            System.err.println(e.getMessage());
```

Comment:
`throw new SkipException(e.getMessage())` to skip this test and not just silently succeed